src/eq_schema/Answer.js

```diff
@@ -1,20 +1,32 @@
 /* eslint-disable camelcase */
+
 class Answer {
   constructor(answer) {
     this.id = "answer-" + answer.id.toString();
     this.mandatory = answer.mandatory;
     this.type = answer.type;
     this.label = answer.label;
 
+    if (answer.type === "Currency") {
+      this.currency = "GBP";
+    }
+
     if (answer.hasOwnProperty("options")) {
-      this.options = answer.options.map(({ label, value, childAnswerId }) => {
-        return {
-          label,
-          value,
-          child_answer_id: childAnswerId || ""
-        };
-      });
+      this.options = answer.options.map(this.buildOption);
+    }
+  }
+
+  buildOption({ label, childAnswerId }) {
+    const option = {
+      label,
+      value: label
```

Comment:
Do we need to transform this value in any way? e.g. making it lower case separated by underscores, kebab case, strip out any special chars?

Reply:
For posterity: no, the value must exactly match the label or the summary screen incorrectly assumes the question was unanswered